File: examples/corpus/go/02_dedup.before.go
package main

func removeDuplicates(arr []int) []int {
	result := []int{}
	for i := 0; i < len(arr); i++ { // remove duplicate elements
		found := false
		for j := 0; j < len(result); j++ {
			if result[j] == arr[i] {
				found = true
			}
		}
		if !found {
			result = append(result, arr[i])
		}
	}
	return result
}

File: examples/corpus/ts/02_dedup.before.ts
function removeDuplicates(arr: number[]): number[] {
  const result: number[] = [];
  for (let i = 0; i < arr.length; i++) { // remove duplicate elements
    if (result.indexOf(arr[i]) === -1) {
      result.push(arr[i]);
    }
  }
  return result;
}

function getFirst(arr: number[]): number {
  const tmp = arr[0];
  return tmp;
}
